src/jdk.jdi/share/classes/com/sun/tools/example/debug/tty/TTYResources.java

```diff
@@ -477,8 +477,9 @@
              "    -dbgtrace [flags] print info for debugging {0}\n" +
              "    -tclient          run the application in the HotSpot(TM) Client Compiler\n" +
              "    -tserver          run the application in the HotSpot(TM) Server Compiler\n" +
+             "    -R<option>        Forward <option> to debuggee process\n" +
```

Comment:
The existing description start with a lowercase character so probably best to keep that consistent.

Reply:
> There is some magic that lets us add "--add-modules", then "jdk.attach" separately, but retrieve them later as "--add-modules=jdk.attach" in the test. I don't see what is doing that, so I ran it and verified the test works.

Yes, there is some magic. No, I don't know where it's done. It seems it must be done by the launcher somewhere, but I can't find it. But the hotspot arguments.cpp does expect it:

```
    } else if (match_option(option, "--add-modules=", &tail)) {
      if (!create_numbered_module_property("jdk.module.addmods", tail, addmods_count++)) {
        return JNI_ENOMEM;
      }
```
I don't see any checks for `--add-modules` without the `=`.

Reply:
ok